include/rdma/fabric.h

```diff
@@ -316,6 +316,23 @@ enum {
 	FI_PROTO_EFA
 };
 
+#define FI_TC_PROV_SPECIFIC (1UL << 31)
+#define FI_TC_DSCP_SET(value) ((value) | FI_TC_DSCP)
+#define FI_TC_DSCP_GET(value) ((value) & ~FI_TC_DSCP)
+
+enum {
+	FI_TC_UNSPEC = 0,
+	FI_TC_DSCP = 0x100,
+	FI_TC_LABEL = 0x200,
+	FI_TC_BEST_EFFORT = FI_TC_LABEL,
+	FI_TC_LOW_LATENCY,
+	FI_TC_DEDICATED_ACCESS,
+	FI_TC_BULK_DATA,
+	FI_TC_SCAVENGER,
+	FI_TC_NETWORK_CTRL,
+	FI_TC_REALTIME_STREAM, /* TODO: pending description @shefty */
```

Comment:
we can drop this until there's a definition for it

Reply:
done